quesma/quesma/router.go

```diff
@@ -425,6 +453,26 @@ func putIndexResult(index string) *mux.Result {
 	return &mux.Result{StatusCode: http.StatusOK, Body: string(serialized)}
 }
 
+func getIndexMappingResult(index string, mappings map[string]any) *mux.Result {
+	result := map[string]any{
+		index: map[string]any{
+			"mappings": mappings,
+		},
+	}
+	serialized, err := json.Marshal(result)
+	if err != nil {
+		panic(err)
```

Comment:
let's avoid using panic's at all cost. 
Throwing 500 will be more appropriate and will keep Quesma going

Reply:
+1

Reply:
Fixed in https://github.com/QuesmaOrg/quesma/pull/658/commits/97bfee9cc80fb958042feb871a4bc932844af234

(also fixed all other `panic`s in that file - all shared the same pattern)